### PR TITLE
Fix CMake for eigenpy 2.7.10

### PR DESCRIPTION
**Pull request by nim65s**

### CODE REVIEW

**Review by jcarpent — APPROVED** [review]

**Comment by cmastalli** [comment]
Does the submodule update requires an update in eigenpy?
Or is there any reason to do so?

**Comment by nim65s** [comment]
the submodule update is not required here.
The main issue is that we now have `eigenpyConfig.cmake` correctly calling `FINDPYTHON()` /`FIND_NUMPY()` automatically for us, but therefore if we try to call it manually after or before eigenpy, it will raise an error.

**Review by cmastalli — APPROVED** [review]

**Comment by nim65s** [comment]
@wxmerkt : can we turn off ROS CI Format job ? we decided not to reformat everything in crocoddyl until we stabilze  the API for v2

**Comment by wxmerkt** [comment]
Sure, happy to turn it off for now

**Review by wxmerkt — APPROVED** [review]
CI is now green. Note that I had to turn the Bionic/18.04 builds with Python2 off as it could not find Python without explicitly specifying PYTHON_EXECUTABLE.

File: dependencies.rosinstall [hunk 1]
```diff
@@ -1,4 +1,4 @@
 - git:
     uri: https://github.com/Gepetto/example-robot-data.git
-    version: master
+    version: devel
```
Comment (cmastalli) [inline dependencies.rosinstall:3]: Is there a reason why we need to use this devel branch?

**Comment by cmastalli** [comment]
> CI is now green. Note that I had to turn the Bionic/18.04 builds with Python2 off as it could not find Python without explicitly specifying PYTHON_EXECUTABLE.

@nim65s do you know how to tackle this? Or are we planning to do not support 18.04/Python 2 anymore?

**Comment by nim65s** [comment]
Don't know. Let's try with just a submodule update

**Comment by nim65s** [comment]
Looks like an issue in example-robot-data

File: dependencies.rosinstall (hunk 1 above)
Comment (wxmerkt) [reply dependencies.rosinstall:3]: Because we need a fix from the devel branch until there is another example-robot-data release

**Comment by cmastalli** [comment]
Thanks guys @nim65s and @wxmerkt :)

**Pull request merged**